src/ir_Vestel.h

```diff
@@ -54,8 +54,23 @@ const uint8_t kVestelACTurbo = 7;
 const uint8_t kVestelACIon = 4;
 const uint8_t kVestelACSwing = 0xA;
 
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wpacked-bitfield-compat"
+const uint8_t kVestelACChecksumOffset = 12;
+const uint8_t kVestelACSwingOffset = 20;
+const uint8_t kVestelACTurboSleepOffset = 24;
+const uint8_t kVestelACTempOffset = 36;
+const uint8_t kVestelACFanOffset = 40;
+const uint8_t kVestelACModeOffset = 44;
+const uint8_t kVestelACIonOffset = 50;
+const uint8_t kVestelACPowerOffset = 52;
+const uint8_t kVestelACOffTimeOffset = 20;
+const uint8_t kVestelACOnTimeOffset = 28;
+const uint8_t kVestelACHourOffset = 36;  // 5 bits
+const uint8_t kVestelACOnTimerFlagOffset = kVestelACHourOffset + 5;
+const uint8_t kVestelACOffTimerFlagOffset = kVestelACHourOffset + 6;
+const uint8_t kVestelACTimerFlagOffset = kVestelACHourOffset + 7;
+const uint8_t kVestelACMinuteOffset = 44;
+
+/*
```

Comment:
Do we need to keep this here as comment?

Reply:
Yeah, I was lazy. I've removed it and added a detailed comment describing the bit-wise breakdown of the message.

Thanks for the good feedback.